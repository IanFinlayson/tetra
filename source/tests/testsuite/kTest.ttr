def main():
   x = 3
   print("In main(), just set x = 3 and x = ", x, "\n")

def main2():
   x = 2
   print("In main2(), this should never happen.")

def main3(x int, y int, s string, q bool, f real, t int):
   x = 3
   print("In main3(), this should never happen.")

