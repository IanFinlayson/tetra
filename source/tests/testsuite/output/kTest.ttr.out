In main(), just set x = 3 and x = 3

